src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
I don't like the idea of having to set it on the Config. That means that the user of the implementation has to always set it to the appropriate one for that implementation (or the implementation has to magically set it invisibly on an object created and provided by the coder when using the Config). Changing objects provided by external code for our own convenience seems very wrong to me. If you change the above statement so that Logics will use a ProfileManager provided by an injected profile manager factory (as we have at present) if none is provided in config, then I'm on board. That way the implementation can specify its default profile manager easily, without modifying objects created by external code, but any coder can choose to use a different profile manager via the config override or the filter override.